### PR TITLE
[OBSDEF-6299] add DefaultRolloverStrategy for s3 logs where it is missing

### PR DESCRIPTION
**Pull request by bst525**
## Purpose
[OBSDEF-6299](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-6299)
Updated dataheadsvc-log4j2.xml with  DefaultRollowverStrategy for access-logs policy

## PR checklist
- [ ] make test passed
- [x] make build passed
- [x] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
_Paste the output of your helm install/vSphere7 deployment testing here_
make test fails - but it fails on master as well
```
Error: fork/exec /root/.local/share/helm/plugins/helm-unittest/untt: no such file or directory
Makefile:67: recipe for target 'test' failed
make: *** [test] Error 1
```

### CODE REVIEW

**Review by SeemaTahaliyani — APPROVED** [review]

File: docs/index.yaml [hunk 1]
```diff
@@ -3048,15 +3048,15 @@ entries:
   influxdb-operator:
   - apiVersion: v2
     appVersion: 0.68.2
-    created: "2021-03-12T14:33:56.372237836-08:00"
+    created: "2021-03-15T15:26:22.347636174-04:00"
     dependencies:
     - name: common-monitoring-lib
       repository: https://asdrepo.isus.emc.com/artifactory/objectscale-helm-build/
-      version: 3.7.0-1142.e856b0d7
+      version: 3.7.0-1137.4d29996c
```
Comment (hldnova) [inline docs/index.yaml:3055]: Is this intended? and is the whole file supposed to be updated?

File: docs/index.yaml [hunk 2]
```diff
@@ -3048,15 +3048,15 @@ entries:
...
     description: |
       InfluxDB Operator deploys operator pod which is
       responsible for managing Influxdb custom resources
-    digest: cf6e4a8ad5e22a93152b028dec0af94ff31d21b4fa7348e069f095756bb92c4e
+    digest: c37aaae9bfebe323a896dd576faee9caee2ceed4aafcee3d964a6ff1c915cef5
```
Comment (hldnova) [inline docs/index.yaml:3059]: same here.

File: docs/index.yaml (hunk 1 above)
Comment (bst525) [reply docs/index.yaml:3055]: I ran " make build CHARTS=ecs-cluster" and added only the changed files.

File: docs/index.yaml (hunk 2 above)
Comment (bst525) [reply docs/index.yaml:3059]: I ran " make build CHARTS=ecs-cluster" and added only the changed files.

File: docs/index.yaml (hunk 1 above)
Comment (eddiepavkovic) [reply docs/index.yaml:3055]: its because the code isn't merged from master.

Comment (bst525) [reply docs/index.yaml:3055]: I cloned from master, created my branch, made the change then ran make build CHARTS=ecs-cluster
My branch came directly from master and my change/build was within 5 minutes of the cloning.

**Review by eddiepavkovic — APPROVED** [review]
its good.  for some reason the auto-monitoring merge messed up the index.yaml.  it has nothing to do with barbs change.

**Review by hldnova — APPROVED** [review]

**Pull request merged**